manifest.yml

```diff
@@ -12,7 +12,7 @@ applications:
   - python_buildpack
   timeout: 180
   services:
-  - PostgreSQL
+  - ElephantSQL
```

Comment:
we also need modification to connect to this database in the cloud. i would checkout init_db https://github.com/rofrano/lab-bluemix-cf/blob/master/service/models.py and slide 138 of 07-PlatformAsAService

Reply:
> we also need modification to connect to this database in the cloud. i would checkout init_db https://github.com/rofrano/lab-bluemix-cf/blob/master/service/models.py and slide 138 of 07-PlatformAsAService

Ok, I'm editing the runtime environment now. I will start another PR.

Reply:
> we also need modification to connect to this database in the cloud before we merge this pull request. i would checkout init_db https://github.com/rofrano/lab-bluemix-cf/blob/master/service/models.py and slide 138 of 07-PlatformAsAService

I looked into it, and I found that the code is very Cloudant specific. I tried a few times but I can't make the connection.

I remember the professor said that he will give us some code that uses ElephantSQL that we can "borrow". So maybe we should wait for the professor?

Reply:
I'm not sure that any additional code would be supplied because it's from a few lectures ago. Maybe there's some additional advice/some additional code referenced in the small part of the lecture that covered connecting to the db in the cloud

Reply:
sorry one more thing - I think that the goal is that we only connect to the ElephantSQL database once we've pushed to the cloud, so I don't think we should be able to connect to the ElephantSQL db until we start execuding the ibm cloud push command

Reply:
I modified the config.py where database's URI will be overrided.
And i guess ElephnatSQL only need URI(containing User and Password and Server) to establish connection?
If that so, I think modifying config.py should be enough.

Reply:
I think in slide 138 of 07-PlatformAsAService it's suggesting that you'll just access the database URI/credentials from vcap_services though so I guess there may not be a need for explicit handling of user, password, and server in our code

Reply:
Yes, I think so, too.
So I just close this pr and start another one.